experiment/prepare.py

```diff
@@ -38,35 +101,26 @@ def make_scenario_dirs(self):
 
         A final function should loop thru N iterations and transform params in each scenario N times.
         """
+
         for cprob in self.experiment_params["IV_levels"]["counterfeit_prob"]:
             for dvar in self.experiment_params["IV_levels"]["dose_variability"]:
                 for fprob in self.experiment_params["IV_levels"]["fentanyl_prob"]:
-                    for user_type, user_params in self.experiment_params[
-                        "user_types"
-                    ].items():
 
-                        scenario_dir = Path(
-                            f"experiment/scenarios/counterfeit_prob_{cprob}/dose_var_{dvar}_fent_prob_{fprob}/{user_type}"
-                        )
-                        scenario_dir.mkdir(parents=True, exist_ok=True)
+                    scenario_dir = Path(
+                        f"experiment/scenarios/counterfeit_prob_{cprob}/dose_var_{dvar}_fent_prob_{fprob}"
+                    )
+                    scenario_dir.mkdir(parents=True, exist_ok=True)
 
-                        scenario_params = deepcopy(self.default_params)
+                    scenario_params = deepcopy(self.default_params)
 
-                        scenario_params["counterfeit_prob"] = cprob
-                        scenario_params["dose_variability"] = dvar
-                        scenario_params["fentanyl_prob"] = fprob
+                    scenario_params["counterfeit_prob"] = cprob
+                    scenario_params["dose_variability"] = dvar
+                    scenario_params["fentanyl_prob"] = fprob
 
-                        scenario_params["starting_dose"] = user_params["starting_dose"]
-                        scenario_params["dose_increase"] = user_params["dose_increase"]
-                        scenario_params["behavioral_variability"] = user_params[
-                            "behavioral_variability"
-                        ]
-                        scenario_params["availability"] = user_params["availability"]
-                        scenario_params["internal_risk"] = user_params["internal_risk"]
-                        scenario_params["external_risk"] = user_params["external_risk"]
+                    with open(scenario_dir.joinpath("params.json"), "w") as f:
+                        json.dump(scenario_params, f)
 
-                        with open(scenario_dir.joinpath("params.json"), "w") as f:
-                            json.dump(scenario_params, f)
+                    #print(scenario_params)
 
     def draw_covariate_values(self):
```

Comment:
Can get rid of this function and `draw_seeds` now that this functionality is in `make_scenario_df`